tests/src/tools/ReadyToRun.SuperIlc/Commands/CompileFromCrossgenRspCommand.cs

```diff
@@ -0,0 +1,271 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace ReadyToRun.SuperIlc
+{
+    class CompileFromCrossgenRspCommand
+    {
+        /// <summary>
+        /// Utility mode that allows compilation of a set of assemblies using their existing Crossgen response files.
+        /// This is currently useful for workloads like Bing which have a large complicated web of binaries in different folders
+        /// with potentially different sets of reference paths used for different assemblies.
```

Comment:
Crossgen's command line interface is weird. E.g. it generates debug information in two passes (because one invokes crossgen once to create the native image, and then one invokes it a second time with `/CreatePDB` to create the PDB). I don't think we should be shooting for a drop in replacement. We can do better than that.

Crossgen command line argument style also can't decide how to spell out the arguments and it's inconsistent with itself - e.g. `/MissingDependenciesOK` and `/App_Ni_Paths`.

Prefixing arguments with `/` is not good for crossplat because then we need to special case Unix.

We started making crossgen's command line more sane by e.g. adding the `/r` switch to specify a referenced assembly (instead of the argument that sucks in all DLLs in the input directory and then causes support cost because people have junk in the directory - the sucking in was causing support cost in .NET Native whenever someone was integrating ilc into their build, and I assume we had similar reasons to add the new argument in crossgen).